caffe2/opt/optimizer.h

```diff
@@ -0,0 +1,17 @@
+#ifndef CAFFE2_OPT_OPTIMIZER_H
+#define CAFFE2_OPT_OPTIMIZER_H
+
+#include "caffe2/core/common.h"
+#include "caffe2/proto/caffe2.pb.h"
+#include "caffe2/core/workspace.h"
+
+namespace caffe2 {
+namespace opt {
+
+NetDef optimize(NetDef net, Workspace* ws, int level = 1);
```

Comment:
If you want to have level, the default should be something like `std::numeric_limits<int>::max()` or `INT_MAX`.

Reply:
I think 1 is safe for now.  there is the issue of optimizations potentially taking too long to apply in the future.